rclcpp_lifecycle/src/lifecycle_node_interface_impl.hpp

```diff
@@ -350,6 +350,18 @@ class LifecycleNode::LifecycleNodeInterfaceImpl
 
   std::vector<Transition>
   get_available_transitions()
+  {
+    std::vector<Transition> transitions;
+    transitions.reserve(state_machine_.current_state->valid_transition_size);
+
+    for (unsigned int i = 0; i < state_machine_.current_state->valid_transition_size; ++i) {
```

Comment:
I know this is because the data structure in C uses `unsigned int`, but really we should be using `size_t` here and there. Nothing to do for now, just a comment...

Reply:
Totally agreed